rental.rb

```diff
@@ -0,0 +1,24 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- Also, you want to be able to access the date, book, and person as the attr_accessor is a shorter way of doing that `attr_accessor :date, :person, :book`. let's kindly update this.